README.md

```diff
@@ -2089,28 +2089,29 @@ have in a company.
 
 In summary, you have to determine or make educated guesses about several things:
 
-- **Equity value**: Determined by the value the company may have in the future, and the number
-  of shares you may own?
+- **Equity value**: This can be estimated by the value the company may have in the future, and
+  the number of shares you may own.
   - **Percentage ownership**: As we’ve mentioned, knowing how many shares of stock or stock
-    options you have is meaningless unless you know the number of outstanding shares.
-    What matters is the percentage ownership of the company the shares represent, including
-    the [details](#counting-shares) of how the total is counted.
-  - **Risk**: It is critical to understand [risk](#stages-of-a-startup) in the business and dilution
-    to ascertain the possible future value of equity for the company and its stage.
+    options you have is meaningless unless you know the number of the company’s outstanding
```

Comment:
Leave as "number of outstanding shares".

Unnecessary, and "the company's outstanding shares" could be misinterpreted.